spec/state_machines/foi_event_spec.rb

```diff
@@ -1,1525 +1,1561 @@
 require 'rails_helper'
 
 describe 'state machine' do
```

Comment:
What has actually changed here?

Reply:
It's more evident if you hide whitespace in the options in GitHub. I moved the current tests into a context called "the usual suspects" and created another context that instantiates cases with old-style closure info (`info_held_status_id = nil`) so that I could test the event permissions there, without adding those old-style closure cases to all the existing tests.